content/_includes/post.macros.njk

```diff
@@ -609,3 +632,82 @@ params:
     </ul>
   {% endif %}
 {% endmacro %}
+
+{# @docs
+label: page_nav
+category: pagination
+note: Navigate paginated collections
+params:
+  title:
+    type: string
+    note: Will be hidden, for screen-readers only
+  pagination:
+    type: 11ty pagination object
+    default: none
+  current:
+    type: boolean
+    default: false
+    note: Show the current location in pagination?
+  id:
+    type: string
+    default: ''
+    note: |
+      When repeating the pagination on a page,
+      each instance should get a unique ID
+  level:
+    type: integer [1-6]
+    default: 2
+    note: Heading level
+#}
+{% macro page_nav(title, pagination, current=true, id='', level=2) %}
+{% if pagination %}
+{% set id = ['resource-nav-title', id] | join('-') %}
+<nav class="pagination" aria-labeledby="{{ id }}">
+  {% h level, {
+    'id': id,
+    'class': 'sr-only'
+  } %}
+    {{ title | mdInline | safe }}
+  {% endh %}
+  <p>
+    {% if pagination.href.next %}
+      {{ utility.link_if('« older', pagination.href.next) }}
+    {% endif %}
+    {% if current %}
+      {{ page_current(pagination) }}
+    {% elif pagination.href.next and pagination.href.previous %}
+      |
+    {% endif %}
+    {% if pagination.href.previous %}
+      {{ utility.link_if('newer »', pagination.href.previous) }}
```

Comment:
I played around with this a bit, and felt like it would take some more design considerations to make it look good. I think we should make a new card for it.

Reply:
Yeah, okay -- tho perhaps it's more straightforward now that we've switched the direction, so it could be on the far left? But I'm fine leaving that for a new card.